src/elements/SingleReview.jsx

```diff
@@ -16,35 +17,51 @@ export const SingleReview = ({
     comment_count,
   },
 }) => {
+  const [localVotes, setLocalVotes] = useState(votes);
   const [comments, setComments] = useState([]);
+  const [commentsLoaded, setCommentsLoaded] = useState(false);
+
   useEffect(() => {
-    getCommentsById(review_id).then((serverComments) =>
-      setComments(serverComments)
-    );
+    getCommentsById(review_id).then((responseComments) => {
+      setCommentsLoaded(true);
+      setComments(responseComments);
+    });
   }, []);
 
   return (
     <div className="SingleReview">
-      <h2>{title}</h2>
-      <p>
-        By {owner} at {created_at.split("T")[0]}
-      </p>
+      <h2 className="spreadLine">
+        {title}{" "}
+        <button className="noWrap"
+          disabled={localVotes !== votes}
+          onClick={() => {
+            patchReviewVotesById(review_id);
+            setLocalVotes((currentVotes) => currentVotes + 1);
```

Comment:
There needs to be error handling here to reflect back to the user that the patch votes has failed ( for example if they have no internet connection ), this can achieved by reverting the vote count back to what it previously was if the patch fails.